sklearn/metrics/pairwise.py

```diff
@@ -164,44 +197,32 @@ def euclidean_distances(X, Y=None, Y_norm_squared=None, squared=False,
     """
     Considering the rows of X (and Y=X) as vectors, compute the
     distance matrix between each pair of vectors.
-
     For efficiency reasons, the euclidean distance between a pair of row
     vector x and y is computed as::
-
```

Comment:
Gotcha. I initially thought you meant I added a whole lot of blank lines, which I obviously could not find. But yeah I do see the removed blank lines, and I have no idea how it happened! Sorry about that.